addons/jump-to-def/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Jump to custom block definition",
```

Comment:
```suggestion
  "name": "Go to custom block definition",
```

Reply:
I think it's okay for "jump" and "go" to be used interchangeably. "Jump" is a better summarizing word and "go" is more natural and works best in a verb phrase.